ParkingApplication.api/Models/ParkingModel.cs

```diff
@@ -7,14 +7,14 @@ namespace ParkingApplication.Api.Models;
 
 public class ParkingModel
 {
-    public int CarId { get; set; }
+    public int Id { get; set; }
```

Comment:
don't you need a relationship with the user here?

Reply:
Done!